ci/check-style.sh

```diff
@@ -61,24 +50,41 @@ replace_original_if_changed() {
 
 # Apply cmake_format to all the CMake list files.
 #     https://github.com/cheshirekow/cmake_format
-find . \( "${ignore[@]}" \) -prune -o \
-  \( -name 'CMakeLists.txt' -o -name '*.cmake' \) \
-  -print0 |
-  xargs -0 cmake-format -i
+git ls-files | grep -P '(CMakeLists\.txt|\.cmake)$' | xargs cmake-format -i
 
 # Apply clang-format(1) to fix whitespace and other formatting rules.
 # The version of clang-format is important, different versions have slightly
 # different formatting output (sigh).
-find google/cloud \( -name '*.cc' -o -name '*.h' \) -print0 |
-  xargs -0 clang-format -i
+git ls-files | grep -P '\.(cc|h)$' | xargs clang-format -i
+
+# Apply buildifier to fix the BUILD and .bzl formatting rules.
+#    https://github.com/bazelbuild/buildtools/tree/master/buildifier
+git ls-files | grep -P '\.(BUILD|bzl)$' | xargs buildifier -mode=fix
```

Comment:
Previously we did `BUILD` and `WORKSPACE` files too.

Reply:
Oops. Accidentally lost those. Added back. Thanks.